wdl-format/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "wdl-format"
+version = "0.1.0"
+license.workspace = true
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dependencies]
+wdl-ast = { path = "../wdl-ast", version = "0.7.1", features = ["codespan"] }
+nonempty.workspace = true
+
+[dev-dependencies]
+pretty_assertions = { workspace = true }
```

nit: I'd just do it this way to match the other crates.

```suggestion
pretty_assertions.workspace = true
```